app/user/status/page.js

```diff
@@ -0,0 +1,91 @@
+'use client'
+import * as React from 'react'
+import CourseCard from '@/app/components/UserPage/CourseCard'
+import CoursePagination from '@/app/components/UserPage/CoursePagination'
+import { Box } from '@mui/material'
+
+const courses = [
```

Comment:
I see that you have two courses variable which have the same value.
These are the mock values so you can create a folde mock/course.js and push this variable to that file